app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors \
+    import NotVaccinatedError, NotWearingMaskError, OutdatedVaccineError
```

Comment:
Use brackets for this:
```suggestion
from app.errors import (
    NotVaccinatedError, NotWearingMaskError, OutdatedVaccineError
)
```